tests/functional/logic_test.sh

```diff
@@ -39,6 +39,8 @@ function test_should_validate_a_non_ok_exit_code() {
     return 1
   }
 
+  set +e
```

Comment:
Note (for the record):
- Using `set -e`, the shell will exit immediately if any command exits with a non-zero status.
- Using `set +e`, the script will continue running even if a command fails.